docs/source/methods/method_of_characteristics.rst

```diff
@@ -18,9 +18,9 @@ The method of characteristics (MOC) is a widely used technique for solving parti
 * track area approximation (:ref:`Section 2.9 <track-area-approximation>`)
 * azimuthal planar projection (:ref:`Section 2.10 <projection-azimuthal-plane>`)
 
-The final equations applied in OpenMOC to solve for the FSR source and scalar flux derived in the following sections are summarized below:
+The final equations applied in OpenMOC to solve for the SR source and scalar flux derived in the following sections are summarized below:
 
-**The source in each flat source region**
+**The source in each source region**
```

Comment:
These equations relate to the "flat" source region approximation, so should we not retain that terminology here? Are you thinking of breaking this up into two sections for fixed and linear source approximations?